src/main/java/com/meraphorce/services/UserService.java

```diff
@@ -1,23 +1,112 @@
 package com.meraphorce.services;
 
-import com.meraphorce.models.User;
-import com.meraphorce.respositories.UserRepository;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import java.util.List;
+import com.meraphorce.dto.UserDTO;
+import com.meraphorce.mapper.impl.UserMapper;
+import com.meraphorce.models.User;
+import com.meraphorce.respositories.UserRepository;
 
 @Service
 public class UserService {
 
     @Autowired
     private UserRepository userRepository;
+    
+    private UserMapper mapper = new UserMapper();
+    
+    Logger logger = LogManager.getLogger(UserService.class);
 
-    public User createUser(User user){
-        return userRepository.save(user);
+    public UserDTO createUser(UserDTO user) {
+    	UserDTO newUser = null;
+    	User data = null;
+    	
+    	
+    	try {
+			data = this.userRepository.save(this.mapper.toEntity(user));
+			newUser = this.mapper.toDTO(data);
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			logger.warn("Error to create user --> " + e);
+		}
+    	
+        return newUser;
     }
 
-    public  List<User> getAllUsers(){
-        return userRepository.findAll();
+    public List<UserDTO> getAllUsers() {
+    	List<UserDTO> list = null;
+    	List<User> data = null;
+    	
+    	try {
+			data = this.userRepository.findAll();
+			
+			if(data != null && !data.isEmpty()) {
+				list = data.stream().map(a -> mapper.toDTO(a)).toList();
```

Comment:
Good use of lambda expression. It can be simplified even more like this

map(UserMapper::toDTO)
It is Method reference
Reference-> https://cdinuwan.medium.com/java-method-reference-lambda-expression-and-stream-c001675b2cf6